packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts

```diff
@@ -162,16 +162,13 @@ export class W3mConnectingWcView extends LitElement {
     switch (this.platform) {
       case 'browser':
         return html`<w3m-connecting-wc-browser></w3m-connecting-wc-browser>`
+      case 'web':
+        return html`<w3m-connecting-wc-web></w3m-connecting-wc-web>`
```

Comment:
how are these different?

Reply:
I think the only difference between those is the UI and class properties.

Reply:
Doesn't the `web` need `onRetry` function?

Reply:
I don't think we need that, since it only has "Open" button and not "Try again" button like on mobile.

<img width="367" alt="image" src="https://github.com/user-attachments/assets/08b86608-8559-4dbe-b61e-e149e38e4b71">